Kernel/FileSystem/SysFS/Subsystems/Kernel/GlobalInformation.h

```diff
@@ -28,6 +28,8 @@ class SysFSGlobalInformation : public SysFSComponent {
     virtual ErrorOr<void> refresh_data(OpenFileDescription&) const override;
     virtual ErrorOr<void> try_generate(KBufferBuilder&) = 0;
 
+    virtual bool jailed_process_readable() const { return false; }
```

Comment:
A bool getter like this one should probably be prefixed with `is_` so `is_readable_by_jailed_processes` or `is_jailed_process_readable`, with a bit of a preference towards something like the first one.